lib/ExtUtils/Mkbootstrap.pm

```diff
@@ -34,17 +33,15 @@ sub Mkbootstrap {
     if (-f "${baseext}_BS"){
 	$_ = "${baseext}_BS";
 	package DynaLoader; # execute code as if in DynaLoader
-	local($osname, $dlsrc) = (); # avoid warnings
-	($osname, $dlsrc) = @Config::Config{qw(osname dlsrc)};
-	$bscode = "";
+	($DynaLoader::osname, $DynaLoader::dlsrc) = @Config::Config{qw(osname dlsrc)};
```

Comment:
Any particular reason for getting rid of the `local`?

Reply:
I can put it back in but I'm unconvinced it actually works.

Reply:
Actually putting the local back is a problem since DynaLoader never declares the variables to begin with. IMO the local is unnecessary in the scope we're talking about here.

Reply:
Why would that be a problem?